frontend/util/gtag.js

```diff
@@ -3,4 +3,5 @@ import ReactGA from "react-ga";
 export const initGA = () => {
   ReactGA.initialize("G-EHMXV36KJV");
   ReactGA.pageview(window.location.pathname + window.location.search);
+  ReactGA.ga("send", "pageview", "/mypage");
```

Comment:
Oops. I think this needs the real page address instead of "/mypage"? Or maybe it's not needed since the previous line creates a pageview event? Sorry, I'm not very familiar with Google Analytics' API.

I think we should probably adjust this at some point to use Next's `useRouter` hook, but that's out of scope for this pull request.

Reply:
@antw Would you replace /mypage with www.holontool.nl? Or would you write it in another format? I'm just checking to see if it makes any difference but want to be sure that I have the format right.

Reply:
Looking at [the documentation for the library](https://www.npmjs.com/package/react-ga), I think the entire line `React.ga(...)` is not needed. The line above sends the pageview event to GA; as far as I know we don't need to send a second pageview event.

Reply:
Ok, then I'll just take it out. Thanks.